test/Array_test.jl

```diff
@@ -226,7 +226,7 @@
             @test begin
                 op = Ryy(; θ = π)
                 y = Matrix(Y())
-                Matrix(op) ≈ 1im * reshape(permutedims(reshape(kron(vec(y), vec(y)), 2, 2, 2, 2), (1, 3, 2, 4)), 4, 4)
+                Matrix(op) ≈ -1im * kron(y, y)
```

Comment:
Yay, seems like they finally fixed `kron`. Before it only worked with vectors.

Reply:
Yes! Also, the following equality was wrong, and it should be instead: $R_{yy}(\theta=\pi)=-i Y\otimes Y$, so I fixed that.